go/epochtime/tendermint/tendermint.go

```diff
@@ -131,16 +129,29 @@ func New(ctx context.Context, service service.TendermintService, interval int64)
 		base:     base,
 		epoch:    base,
 	}
-	r.notifier = pubsub.NewBrokerEx(func(ch *channels.InfiniteChannel) {
-		r.RLock()
-		defer r.RUnlock()
 
-		if r.lastNotified == r.epoch {
-			ch.In() <- r.epoch
+	return r, nil
+}
+
+func (t *tendermintBackend) Start(ctx context.Context) error {
```

Comment:
This seems very awkward, why not initialize in `New` (see my other comment about moving tendermint-specific epochtime backend under `go/tendermint` in advance of #1879)?

Reply:
I needed these since currently `New()` needed a reference to tendermint service, and the tendermint service needed a reference to the timesource.

Agreed it will be nicer if i move this into `go/tendermint` and modify it a bit to avoid having this additional method  :+1: